plugins/woocommerce/client/blocks/assets/js/blocks/product-collection/edit/inspector-advanced-controls/utils.tsx

```diff
@@ -3,48 +3,30 @@
  */
 import { useSelect } from '@wordpress/data';
 import { store as blockEditorStore } from '@wordpress/block-editor';
-
-const unsupportedBlocks = [
-	'core/post-content',
-	'woocommerce/mini-cart',
-	'woocommerce/product-search',
-	'woocommerce/product-title',
-	// Classic wrapper blocks
-	'woocommerce/classic-shortcode',
-	'woocommerce/legacy-template',
-	// Legacy filter blocks
-	'woocommerce/active-filters',
-	'woocommerce/price-filter',
-	'woocommerce/stock-filter',
-	'woocommerce/attribute-filter',
-	'woocommerce/rating-filter',
-	// Deprecated product grid blocks
-	'woocommerce/handpicked-products',
-	'woocommerce/product-best-sellers',
-	'woocommerce/product-category',
-	'woocommerce/product-new',
-	'woocommerce/product-on-sale',
-	'woocommerce/product-tag',
-	'woocommerce/product-top-rated',
-];
-
-const supportedPrefixes = [ 'core/', 'woocommerce/' ];
+import { getBlockSupport } from '@wordpress/blocks';
 
 const isBlockSupported = ( blockName: string ) => {
-	// Check for explicitly unsupported blocks
-	if ( unsupportedBlocks.includes( blockName ) ) {
-		return false;
-	}
+	// Client side navigation can be true in two states:
+	// - supports.interactivity === true;
+	// - supports.interactivity.clientNavigation === true;
+
+	const blockSupportsInteractivity = Object.is(
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore it's a valid supports key
+		getBlockSupport( blockName, 'interactivity' ),
+		true
```

Comment:
Optional, what do you think about using [getBlockType](https://developer.wordpress.org/block-editor/reference-guides/data/data-core-blocks/#getblocktype) here instead, we can retrieve the whole `supports` property for assertion. This can help use reduce the ignore comments here.

Reply:
Hmm.. While I'm obviously tempted by removing the ignore comments (!) I believe that should not drive the implementation. `getBlockSupport` does exactly what we need and I think it's actually a beneficial the lack of types is highlighted rather than covered by more high-level function.

_(But it's still tempting 😅 )_